src/runtime_src/core/edge/drm/zocl/zocl_xclbin.c

```diff
@@ -197,6 +198,23 @@ zocl_load_pskernel(struct drm_zocl_dev *zdev, struct axlf *axlf)
 	return 0;
 }
 
+static int
+zocl_check_sect(enum axlf_section_kind kind, struct axlf *axlf_full)
+{
+	uint64_t offset;
+	uint64_t size;
+	int err = 0;
+
+	err = xrt_xclbin_section_info(axlf_full, kind, &offset, &size);
+	if (err) {
+		DRM_WARN("get section %s err: %d ",
+		    xrt_xclbin_kind_to_string(kind), err);
+		return 0;
```

Comment:
We don't really need to print this warning. It is a call generator while nothing is wrong but makes the user nervous. I know there a bunch of this warning in other places but they will be cleaned up in this release.

Reply:
fixed